2022-09-25 09:37:46,337 ERROR 15927 [-/127.0.0.1/-/153ms GET /default/getListById/undefined] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    --------------------
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/client.js:38:20)
    at RDSClient.proto.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:65:19)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:54:5
    sql: SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id WHERE type_id=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id WHERE type_id=undefined"
name: "ER_BAD_FIELD_ERRORError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 09:38:56,117 ERROR 15927 [-/127.0.0.1/-/158ms GET /default/getListById/undefined] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    --------------------
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/client.js:38:20)
    at RDSClient.proto.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:65:19)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:54:5
    sql: SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id WHERE type_id=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id WHERE type_id=undefined"
name: "ER_BAD_FIELD_ERRORError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 09:39:19,977 ERROR 15927 [-/127.0.0.1/-/145ms GET /default/getListById/undefined] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    --------------------
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/client.js:38:20)
    at RDSClient.proto.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:65:19)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:54:5
    sql: SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id WHERE type_id=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id WHERE type_id=undefined"
name: "ER_BAD_FIELD_ERRORError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 09:40:56,643 ERROR 15927 [-/127.0.0.1/-/150ms GET /default/getListById/undefined] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:527:28)
    at addChunk (node:internal/streams/readable:315:12)
    --------------------
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:199:23)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as query] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at RDSClient.proto._query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/client.js:38:20)
    at RDSClient.proto.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/ali-rds/lib/operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (/Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:65:19)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/co/index.js:54:5
    sql: SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id WHERE type_id=undefined
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'undefined' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id WHERE type_id=undefined"
name: "ER_BAD_FIELD_ERRORError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 10:57:04,911 ERROR 15927 [-/127.0.0.1/-/10066ms GET /default/getTypeInfo] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT * FROM `type`
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 10:57:14,924 ERROR 15927 [-/undefined/-/10002ms GET /default/getTypeInfo] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT * FROM `type`
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
headerSent: true
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 10:57:22,310 ERROR 15927 [-/127.0.0.1/-/10003ms GET /default/getTypeInfo] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT * FROM `type`
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 10:57:32,318 ERROR 15927 [-/127.0.0.1/-/10002ms GET /default/getTypeInfo] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT * FROM `type`
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 10:57:42,846 ERROR 15927 [-/127.0.0.1/-/10004ms GET /default/getTypeInfo] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT * FROM `type`
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 10:57:43,137 ERROR 15927 [-/127.0.0.1/-/10025ms GET /default/getArticleList] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 10:57:52,852 ERROR 15927 [-/127.0.0.1/-/10002ms GET /default/getTypeInfo] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT * FROM `type`
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 11:10:38,897 ERROR 15927 [-/127.0.0.1/-/10005ms GET /default/getArticleList] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 11:12:53,230 ERROR 15927 [-/127.0.0.1/-/10005ms GET /default/getArticleList] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 11:12:54,213 ERROR 15927 [-/127.0.0.1/-/10014ms GET /default/getTypeInfo] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT * FROM `type`
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
headers: {"Access-Control-Allow-Origin":"http://localhost:3000","Access-Control-Allow-Credentials":"true","vary":"Origin"}
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 11:13:12,444 ERROR 15927 [-/127.0.0.1/-/10002ms GET /default/getArticleList] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 11:13:15,051 ERROR 15927 [-/127.0.0.1/-/10003ms GET /default/getArticleList] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 11:13:20,176 ERROR 15927 [-/127.0.0.1/-/10004ms GET /default/getArticleList] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: SELECT article.id as id ,article.title as title ,article.intro as intro ,FROM_UNIXTIME(article.addTime,'%Y-%m-%d %H:%i:%s') as addTime ,article.view_count as view_count ,type.typeName as typeName FROM article LEFT JOIN type ON article.type_id = type.id
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
name: "ETIMEDOUTError"
pid: 15927
hostname: hahas-MBP.lan

2022-09-25 11:14:16,448 ERROR 20822 [-/127.0.0.1/-/1ms GET /] nodejs.ETIMEDOUTError: connect ETIMEDOUT
    at PoolConnection.Connection._handleConnectTimeout (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:409:13)
    at Object.onceWrapper (node:events:641:28)
    at Socket.emit (node:events:527:28)
    at Socket._onTimeout (node:net:516:8)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
    --------------------
    at Protocol._enqueue (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/haha/Documents/react/react_blog2/service4/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/haha/Documents/react/react_blog2/service4/node_modules/pify/index.js:29:7
    sql: select now() as currentTime;
errorno: "ETIMEDOUT"
code: "ETIMEDOUT"
syscall: "connect"
fatal: true
name: "ETIMEDOUTError"
pid: 20822
hostname: hahas-MBP.lan

